CHANGELOG.md

```diff
@@ -33,7 +32,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Fixes ingestPdrWithNodeNameSpec parsePdr provider error
 
 ### BREAKING CHANGES
-
+- **CUMULUS-2138** - CMR metadata update behavior has been removed from the `move-granules` task into a 
```

Comment:
Can we move `As a result...` and everything after up into a note under "Migration notes"?

Reply:
Yes